crates/oxc_linter/src/rules/jest/no_disabled_tests.rs

```diff
@@ -153,73 +153,129 @@ fn run<'a>(possible_jest_node: &PossibleJestNode<'a, '_>, ctx: &LintContext<'a>)
 #[test]
 fn test() {
     use crate::tester::Tester;
+    use std::path::PathBuf;
+
+    let jest_path = "/no-disabled-tests.test.ts";
 
     let mut pass = vec![
-        ("describe('foo', function () {})", None),
-        ("it('foo', function () {})", None),
-        ("describe.only('foo', function () {})", None),
-        ("it.only('foo', function () {})", None),
-        ("it.each('foo', () => {})", None),
-        ("it.concurrent('foo', function () {})", None),
-        ("test('foo', function () {})", None),
-        ("test.only('foo', function () {})", None),
-        ("test.concurrent('foo', function () {})", None),
-        ("describe[`${'skip'}`]('foo', function () {})", None),
-        ("it.todo('fill this later')", None),
-        ("var appliedSkip = describe.skip; appliedSkip.apply(describe)", None),
-        ("var calledSkip = it.skip; calledSkip.call(it)", None),
-        ("({ f: function () {} }).f()", None),
-        ("(a || b).f()", None),
-        ("itHappensToStartWithIt()", None),
-        ("testSomething()", None),
-        ("xitSomethingElse()", None),
-        ("xitiViewMap()", None),
+        ("describe('foo', function () {})", None, None, Some(PathBuf::from(jest_path))),
```

Comment:
We don't need to change test cases in rules, you can update path at once:

```diff
let path_to_lint = if self.import_plugin {
      assert!(path.is_none(), "import plugin does not support path");
      self.current_working_directory.join(&self.rule_path)
  } else if let Some(path) = path {
      self.current_working_directory.join(path)
+ } else if self.jest_plugin {
+     self.rule_path.with_extension("test.tsx")
  } else {
      self.rule_path.clone()
  };
```

Reply:
Okay, I'll updated it later.

Reply:
Done.